docs/reference/indices.asciidoc

```diff
@@ -140,3 +141,4 @@ include::indices/split-index.asciidoc[]
 include::indices/apis/unfreeze.asciidoc[]
 include::indices/update-settings.asciidoc[]
 include::indices/put-mapping.asciidoc[]
+include::indices/mode-settings.asciidoc[]
```

Comment:
As a result of this I see the "Index mode" link showing up under the "Rest API" menu. I don't think this is the right way of linking it. Ideally just having the link from the page...see above <<indices-mode-settings>> is enough in my opinion. Is there a way to hide the "Index mode" section which appears under "Rest API"?

Reply:
I think I will put the link in the "Create API" page where we talk about settings.